account_operating_unit/models/account_move.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2016 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp.tools.translate import _
+from openerp import api, fields, models
+from openerp.exceptions import UserError
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    operating_unit_id = fields.Many2one('operating.unit', 'Operating Unit',
+                                        default=lambda self:
+                                        self.env['res.users'].
+                                        operating_unit_default_get(self._uid))
+
+    @api.model
+    def create(self, vals):
+        if vals.get('move_id', False):
+            move = self.env['account.move'].browse(vals['move_id'])
+            if move.operating_unit_id:
+                vals['operating_unit_id'] = move.operating_unit_id.id
+        return super(AccountMoveLine, self).create(vals)
+
+    @api.model
+    def _query_get(self, domain=None):
+        if self._context.get('operating_unit_ids', False):
+            domain.append(('operating_unit_id', 'in',
+                           self._context.get('operating_unit_ids')))
+        return super(AccountMoveLine, self)._query_get(domain)
+
+    @api.one
+    @api.constrains('operating_unit_id', 'company_id')
+    def _check_company_operating_unit(self):
+        if self.company_id and self.operating_unit_id and \
+                self.company_id != self.operating_unit_id.company_id:
+            raise UserError(_('Configuration error!\nThe Company in the\
```

Comment:
The ending slash in not syntax: it's literally in the string.
Just close the string and Python will implicitly concat it with the next line's string.